styleguide/derek/pattern-components/tabs/tabs-horizontal.ejs

```diff
@@ -1,10 +1,10 @@
 <div class="container">
   <div class="rsTabs-container rsTabsHorizontal-container">
     <ul class="nav nav-tabs rsTabs rsTabsHorizontal">
-      <li class="rsTab-tab active"><a class="rsTab-link rsTabHorizontal-link" href="#tab-1" data-toggle="tab">Tab 1</a></li>
-      <li class="rsTab-tab"><a class="rsTab-link rsTabHorizontal-link" href="#tab-2" data-toggle="tab">Tab 2</a></li>
-      <li class="rsTab-tab"><a class="rsTab-link rsTabHorizontal-link" href="#tab-3" data-toggle="tab">Tab 3</a></li>
-      <li class="rsTab-tab"><a class="rsTab-link rsTabHorizontal-link" href="#tab-4" data-toggle="tab">Tab 4</a></li>
+      <li class="rsTab-tab active"><a class="rsTab-link rsTabHorizontal-link" href="#tab-1" data-toggle="tab" event-track>Tab 1</a></li>
```

Comment:
These `event-track` attributes can be removed. Tracking is handled by JS code in www acting on the `.rsTabs` class.